velox/functions/prestosql/tests/GreatestLeastTest.cpp

```diff
@@ -181,3 +190,97 @@ TEST_F(GreatestLeastTest, clearNulls) {
     }
   }
 }
+
+TEST_F(GreatestLeastTest, shortDecimal) {
+  const auto shortType = DECIMAL(10, 4);
+  const std::vector<std::optional<int64_t>> firstColumnUnscaled = {
+      10000,
+      -10000,
+      20000,
+      UnscaledShortDecimal::max().unscaledValue() - 1,
```

Comment:
Would introducing local constants kMin and kMax help make this code shorter and easier to read?

Reply:
Yes it helps make the code more concise, thanks for the suggestion!